### PR TITLE
Feat/484317  update owner responsive

### PR DESCRIPTION
**Pull request by whitewaterdesign**

### CODE REVIEW

**Review by jbarnsley10 — APPROVED** [review]
LGTM but obvs don;t merge until after PRE

File: app/frontend/css/application.scss [hunk 1]
```diff
@@ -101,6 +101,14 @@ $govuk-global-styles: true; // https://frontend.design-system.service.gov.uk/sas
     }
   }
 
+  &-margin-bottom-9 {
+
+    @media (max-width: 40.0625rem) {
```
Comment (jbarnsley10) [inline app/frontend/css/application.scss:106]: Was the 40.0625rem not set as a global (something like desktop-width)?

Comment (whitewaterdesign) [reply app/frontend/css/application.scss]: updated

File: app/views/cdo/manage/cdo.njk [hunk 2]
```diff
@@ -47,23 +48,23 @@
...
-
 
       <div class="govuk-grid-column-three-quarters">
 
       <h2 class="govuk-heading-m govuk-!-margin-bottom-2">
         CDO progress
       </h2>
 
-      {{ govukTaskList(model.taskList) }}
+      {{ govukSummaryList(model.summaryList) }}
```
[the remark this answers is not in the document]
Comment (whitewaterdesign) [reply app/views/cdo/manage/cdo.njk:66]: should be sorted now

**Pull request merged**